worlds/alttp/Rom.py

```diff
@@ -37,7 +38,7 @@
 from Utils import local_path, user_path, int16_as_bytes, int32_as_bytes, snes_to_pc, is_frozen
 from worlds.alttp.Items import ItemFactory, item_table, item_name_groups, progression_items
 from worlds.alttp.EntranceShuffle import door_addresses
-from worlds.alttp.Options import smallkey_shuffle
+from worlds.alttp.Options import smallkey_shuffle, PrizeShuffle
```

Comment:
smallkey_shuffle should be renamed to SmallkeyShuffle, but i guess that can be done separately